code/style.css

```diff
@@ -1,5 +1,129 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
 
 body {
-  background-color: lightblue;
+ margin: 0;
+ font-family: 'Montserrat', sans-serif;
 }
+
+header {
+  margin: 0px;
+  padding-top: 5px;
+  background-image: url("images/Fraumitstoff.jpg");
+  background-size: cover;
+  height: 1220px;
+  }
+
+h1 {
+  
+  margin-top: 40px;
+  text-align: center;
+  align-items: center;
+  font-size:90px;
+}
+
+.styled-button {
+  font-family: monospace;
+  font-size: 20px;
+
+  padding: 5px;
+
+  color: black;
+  background: white;
+
+  border: 2px solid teal;
+  border-radius: 10px;
+
+  cursor: pointer;
+  width: 10cm;
+  height: 1.5cm;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.styled-button:hover {
+  border: 2px solid thistle;
+  border-radius: 10px;
+}
+
+.styled-button {
+  border: 2px solid teal;
+  border-radius: 10px;
+}
+
+h2 {
+  font-size: 30px;
+  align-items: center;
+}
+
+
+menu {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  float: right;
+}
+
+menu a {
+  display: inline-block;
+  width: 4cm;
+  text-align: center;
+  padding: 10px 20px;
+  color: black;
+  font-weight: bold;
+  text-decoration: none;
+  border-radius: 5px;
+  margin: 0 5px;
+}
+
+menu a:hover {
+  background: #8FBC8F;
+  color: white;
+}
+
+.grid-container {
+  border: 10px solid white;
+  height: 500px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  font-size: 30px;
+  text-align: center;
+  grid-gap: 10px;
+  color: black;
+}
+
+.grid-item1 {
+  background-color: transparent;
+}
```

Comment:
A suggestion to shorten the code below is to use something like:
`.grid-item1, .grid-item2, .grid-item3, .grid-item4 {`
`  background-color: transparent;`
`}`